app/views/topics/show.html.erb

```diff
@@ -6,21 +6,16 @@
 
       <div class="cards">
         <div class="card w-100">
-          <img src="https://images.unsplash.com/photo-1561451213-d5c9f0951fdf?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1489&q=80" class="card-img-top" alt="...">
           <div class="card-body">
-            <h5 class="card-title"><%= tweet['user']['username']%></h5>
+            <h5 class="card-title">@<%= tweet['user']['username']%> </h5>
+            <h6><%= number_to_human(tweet['user']['public_metrics']['followers_count'], units: { thousand: 'K', million: 'M' }) %> Followers </h6>
             <p class="card-text"><%= tweet['text'] %></p>
             <ul class="d-flex justify-content-around p-0">
               <li class="list-unstyled">
-                <%= link_to root_path do %>
+                <%= link_to "https://twitter.com/#{tweet['user']['username']}/status/#{tweet['id']}", target: '_blank' do %>
```

Comment:
Would be good to have it refactored as a helper
https://github.com/andrerferrer/rails-helper-demo